src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate(opetation, number) {
```

Comment:
Let`s rename first parametr to "operation". And use .call() to call your callback with provided context and arguments.

Reply:
Oh, nice! I didn't know about this method)